### PR TITLE
Sync service ca

### PR DESCRIPTION
**Pull request by ericavonb**
for after https://github.com/openshift/service-ca-operator/pull/19

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -77,6 +86,20 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
...
+	resourceSyncController := resourcesynccontroller.NewResourceSyncController(
+		operatorClient,
+		kubeInformersForNamespaces,
+		v1helpers.CachedSecretGetter(kubeClient.CoreV1(), kubeInformersForNamespaces),
+		v1helpers.CachedConfigMapGetter(kubeClient.CoreV1(), kubeInformersForNamespaces),
+		ctx.EventRecorder,
+	)
+	if err := resourceSyncController.SyncConfigMap(
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "service-ca"},
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.TargetNamespace, Name: api.SigningCABundleConfigMapName},
```
Comment (enj) [inline pkg/operator/starter.go]: I still want to move the source into `OperatorNamespace` but we can followup.

Comment (ericavonb) [reply pkg/operator/starter.go]: exactly my plan

**Pull request merged**